exercises/settings.gradle

```diff
@@ -101,3 +101,5 @@ include 'word-count'
 include 'word-search'
 include 'wordy'
 include 'zipper'
+include 'grep'
```

Comment:
Are these exercises meant to be alphabetised? :slightly_smiling_face:

Reply:
Yes they are, thanks for reminding me! Will move `grep` so that it's in the correct order 🙂